src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1573,6 +1588,128 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
 
         }
 
+        /// <inheritdoc/>
+        [Obsolete]
+        public async override Task RaiseWorkflowEventAsync(
+            string instanceId,
+            string workflowComponent,
+            string eventName,
+            Object eventData,
+            CancellationToken cancellationToken = default)
+        {
+            ArgumentVerifier.ThrowIfNullOrEmpty(instanceId, nameof(instanceId));
+            ArgumentVerifier.ThrowIfNullOrEmpty(workflowComponent, nameof(workflowComponent));
+            ArgumentVerifier.ThrowIfNullOrEmpty(eventName, nameof(eventName));
+
+            // Serialize json data. Converts eventData object to bytes and then bytestring inside the request.
+            byte[] jsonUtf8Bytes = JsonSerializer.SerializeToUtf8Bytes(eventData);
```

Comment:
I think we need to check for null here. Otherwise, `JsonSerializer.SerializeToUtf8Bytes(eventData)` will return a 4-byte response for "null", which is probably unexpected.